frontend/apps/reader/modules/readerfont.lua

```diff
@@ -44,6 +46,17 @@ function ReaderFont:init()
                 event = "ChangeLineSpace", args = "decrease" },
         }
     end
+    if Device:isTouchDevice() then
+        local range = Geom:new{
+            x = 0, y = 0,
+            w = Screen:getWidth(),
+            h = Screen:getHeight(),
+        }
+        self.ges_events = {
```

Comment:
It's better to use the new touch zone feature, see:
https://github.com/koreader/koreader/blob/master/frontend/apps/reader/modules/readerpaging.lua#L79
http://koreader.rocks/doc/modules/ui.widget.container.inputcontainer.html#InputContainer:registerTouchZones

Touch zone will automatically update gesture range for you on screen resize.